lms/djangoapps/commerce/api/v1/models.py

```diff
@@ -22,6 +22,19 @@ def __init__(self, id, modes):  # pylint: disable=invalid-name,redefined-builtin
         self.modes = list(modes)
         self._deleted_modes = []
 
+    @property
+    def name(self):
+        """ Return course name. """
+        course_id = CourseKey.from_string(unicode(self.id))  # pylint: disable=invalid-name
+
+        try:
+            return CourseOverview.get_from_id(course_id).display_name
+        except CourseOverview.DoesNotExist:
+            # NOTE (CCB): Ideally, the course modes table should only contain data for courses that exist in
+            # modulestore. If that is not the case, say for local development/testing, carry on without failure.
+            log.warning('Failed to retrieve CourseOverview for [%s]. Using empty course name.', course_id)
+            return None
```

Comment:
will an empty name break Otto?   (if so, maybe put something generic in there, or echo the course id...if not, ignore me)